app/src/main/java/io/transcend/samplesdk/ManageConsentPreferences.java

```diff
@@ -47,12 +29,10 @@ protected void onCreate(Bundle savedInstanceState) {
         // Example: here I want to change the destroy on close behavior to false
         TranscendCoreConfig config = TranscendAPI.config.clone();
         config.setDestroyOnClose(false);
-        System.out.println(TranscendAPI.config.getDestroyOnClose());
```

Comment:
i dunno java but this looks like removing a print statement? why is this being removed

Reply:
Just clearing out unnecessary logging